lua/core/info.lua

```diff
@@ -65,44 +66,55 @@ local function tbl_set_highlight(terms, highlight_group)
   end
 end
 
+local function make_client_info(client)
+  local client_enabled_caps = require("lsp.utils").get_ls_capabilities(client.id)
+  local name = client.name
+  local id = client.id
+  local document_formatting = client.resolved_capabilities.document_formatting
+  local client_info = {
+    fmt("* Name:                 %s", name),
+    fmt("* Id:                   %s", tostring(id)),
+    fmt("* Supports formatting:  %s", tostring(document_formatting)),
+  }
+  if not vim.tbl_isempty(client_enabled_caps) then
+    local caps_text = "* Capabilities list:    "
+    local caps_text_len = caps_text:len()
+    local enabled_caps = text.format_table(client_enabled_caps, 3, " | ")
+    enabled_caps = text.shift_right(enabled_caps, caps_text_len)
+    enabled_caps[1] = fmt("%s%s", caps_text, enabled_caps[1]:sub(caps_text_len + 1))
+    vim.list_extend(client_info, enabled_caps)
+  end
+
+  return client_info
+end
+
 function M.toggle_popup(ft)
   local lsp_utils = require "lsp.utils"
-  local client = lsp_utils.get_active_client_by_ft(ft)
-  local is_client_active = false
-  local client_enabled_caps = {}
-  local client_name = ""
-  local client_id = 0
-  local document_formatting = false
-  if client ~= nil then
-    is_client_active = not client.is_stopped()
-    client_enabled_caps = require("lsp").get_ls_capabilities(client.id)
-    client_name = client.name
-    client_id = client.id
-    document_formatting = client.resolved_capabilities.document_formatting
-  end
+  local clients = lsp_utils.get_active_client_by_ft(ft)
+  local client_names = {}
 
   local header = {
     fmt("Detected filetype:      %s", ft),
     fmt("Treesitter active:      %s", tostring(next(vim.treesitter.highlighter.active) ~= nil)),
   }
 
-  local text = require "interface.text"
   local lsp_info = {
     "Language Server Protocol (LSP) info",
-    fmt("* Associated server:    %s", client_name),
-    fmt("* Active:               %s (id: %d)", tostring(is_client_active), client_id),
-    fmt("* Supports formatting:  %s", tostring(document_formatting)),
+    fmt "* Associated server(s):",
```

Comment:
You removed the parameter, the server is not displayed anymore.
Also, it might be useful to print whether the server is active or not.

Reply:
I'm not sure if this is exactly what you mean, but "active" parameter is redundant in this case since there's no a priori info about the server because that info is not in the global state anymore. 
So the way it works [now] is that if it's active, then it'll show up.

Reply:
Alright, then only the name of the LS is missing